src/Tizen.Applications.EventManager/Tizen.Applications/SystemEvents.cs

```diff
@@ -0,0 +1,905 @@
+﻿/*
+ * Copyright (c) 2019 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+namespace Tizen.Applications.EventManager
+{
+    /// <summary>
+    /// Supported System Events
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public static class SystemEvents
```

Comment:
Why not a namespace?

Reply:
What do you mean? Should the class name be the same as the namespace?

Reply:
You have multiple nested static classes inside yet another static class. I assume you want to hide the access to those nested classes. However, this outer static class has no fields or methods, which means that it could be just another level of namespace like 
`using namespace Tizen.Applications.EventManager.SystemEvents;` 
`var x = BatteryChargerStatus.EventName;` 
instead of 
`var x = SystemEvents.BatteryChargerStatus.EventName;`
In second one you need to repeat the `SystemEvents` part each time you use any of the nested classes. This question is concerned about the matter of convenience and not about a proper implementation of that issue.